### DISCUSSION TITLE
new layer2 of bitcoin with big backers

### DISCUSSION THREAD

**Discussion by jayjose1** [b2network/docs#72] [Show and tell]
Bitcoin, as the primary crypto asset, can evolve beyond a payment network.
https://task.bsquared.network/leaderboard?code=lPzxB9o
Fast, Cheap & Secure

B² Network is over 50x cheaper and 300x faster than the Bitcoin network.

B² Network employs Bitcoin-level security with Bitcoin POW (L1) & Zero-Knowledge Proof (L2).
BTC-related assets require an ecosystem where DeFi, NFT, SocialFi, and other applications can thrive and flourish.

**Comment by teng-tt** [comment]
The B2 test network node is so stuck. It has been 5 days since tBTC to B2 was synchronized successfully. Is there any problem?
